axelrod/moran.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from collections import Counter
+import random
+
+import numpy as np
+
+from .deterministic_cache import DeterministicCache
+from .match import Match
+from .player import Player
+from .random_ import randrange
+
+
+def fitness_proportionate_selection(scores):
+    """Randomly selects an individual proportionally to score."""
+    csums = np.cumsum(scores)
+    total = csums[-1]
+    r = random.random() * total
+
+    for i, x in enumerate(csums):
+        if x >= r:
+            return i
```

Comment:
I think this can be done simpler with `numpy.random.choice` as in https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/strategy_transformers.py#L296

http://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.random.choice.html